components/extractors/marketing-sales/facebook-ads/index.md

```diff
@@ -44,10 +44,13 @@ The query describes the extractor request to be sent to the Facebook Marketing A
 make creating a query easy because all options except `name` represent the [Facebook Marketing API
 request](https://developers.facebook.com/docs/graph-api/using-graph-api) parameters.
 
+### Async Insights Query
+The *Async Insights Query* option specify the way extractor requests the data. If it is checked then data will be extracted asynchronously, the extractor will make an initial request, then by polling fb api waits for the result and once ready extracts it. This allows for bigger amount of data to be extracted especially when the synchronous extraction ends up with error `Please reduce the amount of data you're asking for`. Note that only insights data of the [ads objects](https://developers.facebook.com/docs/marketing-api/insights/parameters/) can be extracted asynchronously.
```

Comment:
```suggestion
The *Async Insights Query* option specify the way extractor requests the data. If it is checked then data will be extracted asynchronously, the extractor will make an initial request, then by polling fb api waits for the result and once ready extracts it. This allows for bigger amount of data to be extracted especially when the synchronous extraction ends up with error `Please reduce the amount of data you're asking for`. Note that only insights data of the [ads objects](https://developers.facebook.com/docs/marketing-api/insights/parameters/) can be extracted asynchronously.
```

The *Async Insights Query* option specify the way extractor requests the data. If it is checked then data will be extracted asynchronously. The extractor will make an initial request, then by polling the Facebook API it waits for the result and extracts it when it becomes ready. This allows for bigger amount of data to be extracted especially when the synchronous extraction ends up with error `Please reduce the amount of data you're asking for`. Note that only the insights data of the [ads objects](https://developers.facebook.com/docs/marketing-api/insights/parameters/) can be extracted asynchronously.

Reply:
z toho linku ads objects teda není zřejmý na co se vlastně odkazuješ - o insights data se tam nemluví